### PR TITLE
example pull request

### PR DESCRIPTION
**Pull request by the-goat-man**
adding checks, .github, and other stuff

### CODE REVIEW

**Review by Blaukreutz — COMMENTED** [review]
looks good

File: src/main/java/org/recordrobotics/charger/Main.java [hunk 1]
```diff
@@ -12,14 +12,14 @@
  * call.
  */
 public final class Main {
-  private Main() {}
+	private Main() {}
```
Comment (the-goat-man) [inline src/main/java/org/recordrobotics/charger/Main.java:15]: what's the indent for??????????

File: src/main/java/org/recordrobotics/charger/Main.java [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
-   */
-  public static void main(String... args) {
-    RobotBase.startRobot(Robot::new);
-  }
+	/**
+	 * Main initialization function. Do not perform any initialization here.
+	 *
+	 * <p>If you change your main robot class, change the parameter type.
+	 */
+	public static void main(String... args) {
```
Comment (the-goat-man) [inline src/main/java/org/recordrobotics/charger/Main.java:22]: bruh what is this

**Review by the-goat-man — COMMENTED** [review]
imagine formatting

File: build.gradle [hunk 3]
```diff
@@ -1,40 +1,60 @@
 plugins {
-    id "java"
-    id "edu.wpi.first.GradleRIO" version "2022.2.1"
+	id "java"
+	id "pmd"
+	id "edu.wpi.first.GradleRIO" version "2022.3.1"
```
Comment (vladaviedov) [inline build.gradle:4]: Bad version

**Review by vladaviedov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**